### PR TITLE
[ticket/15536] Catalog: adds satis.phpbb.com to repositories

### PR DESCRIPTION
**Pull request by Nicofuma**
Also always installs composer/installers in vendor-ext too.

PHPBB3-15536

Checklist:

- [x] Correct branch: master for new features; 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15536

### CODE REVIEW

File: phpBB/phpbb/db/migration/data/v330/extensions_composer_2.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+*
+*/
+
+namespace phpbb\db\migration\data\v330;
+
+class extensions_composer_2 extends \phpbb\db\migration\migration
+{
+	public function update_data()
+	{
+		$repositories = json_decode($this->config, true);
```
Comment (marc1706) [inline phpBB/phpbb/db/migration/data/v330/extensions_composer_2.php]: Shouldn't this be `json_decode($this->config->get(''exts_composer_repositories''), true)`?

Comment (Nicofuma) [reply phpBB/phpbb/db/migration/data/v330/extensions_composer_2.php]: indeed...

Comment (Nicofuma) [reply phpBB/phpbb/db/migration/data/v330/extensions_composer_2.php]: fixed

**Comment by marc1706** [comment]
@Nicofuma does this depend on anything or can we merge it?

**Comment by Nicofuma** [comment]
can be merged

**Pull request merged**